daemon/api_general.go

```diff
@@ -231,6 +233,25 @@ func getChanges(c *Command, r *http.Request, user *auth.UserState) Response {
 		return BadRequest("select should be one of: all,in-progress,ready")
 	}
 
+	follow := false
+	if s := query.Get("follow"); s != "" {
+		f, err := strconv.ParseBool(s)
+		if err != nil {
+			return BadRequest(`invalid value for follow: %q: %v`, s, err)
```

Comment:
If `BadRequest` is anything like `fmt.Errorf` you may want to use `%w` instead of `%v`